build-parent/pom.xml

```diff
@@ -863,6 +864,9 @@
                             <TESTCONTAINERS_RYUK_DISABLED>${TESTCONTAINERS_RYUK_DISABLED}</TESTCONTAINERS_RYUK_DISABLED>
                             <!--<TESTCONTAINERS_HUB_IMAGE_NAME_PREFIX>YOUR MIRROR OR INTENRAL REPO GOES HERE - e.g., nexus.foobar.com/</TESTCONTAINERS_HUB_IMAGE_NAME_PREFIX>-->
                         </environmentVariables>
+                        <!-- Spark needs to use an internal API that has been made inaccessible by default. This
+                        exposes it -->
+                        <argLine>--add-exports java.base/sun.nio.ch=ALL-UNNAMED</argLine>
```

Comment:
Q/S: Do we need to add something similar for `failsafe`?

Reply:
No integration tests or use of failsafe in the extensions-data-delivery module where this was causing failures. Does not hurt to add though and I think it could save us time in the future